lms/resources/_lti_launch.py

```diff
@@ -183,6 +182,45 @@ def js_config(self):
 
         :rtype: dict
         """
+
+        def hypothesis_client_config():
```

Comment:
I left this as an internal function, within the `LTILaunchResource.js_config()` method. Later, when all this config stuff moves into a separate service, we'll probably want to move each block of config into a separate method (perhaps private helper methods) and have one master method that stitches them all together. Just to avoid having one really big method. But for now this will do.

Reply:
As far as I can tell this function doesn't form a closure around anything and so doesn't _need_ to be inline. What's the motivation for an inline function, rather than a private function?

Reply:
As far as I can tell this function doesn't form a closure around anything and so doesn't need to be inline. What's the motivation for an inline function, rather than a private function?

Reply:
Yep. Sometimes if a function only exists to be a private helper of one other function it can be readable to make it an inner function just for that reason, even if no closure is used. In retrospect though this function is a bit long to be an inner function. Moved it back out and made it a private method instead